test-browser/tests/remixd.js

```diff
@@ -52,15 +52,20 @@ module.exports = {
       when a relative import is used (i.e import "openzeppelin-solidity/contracts/math/SafeMath.sol")
       remix (as well as truffle) try to resolve it against the node_modules and installed_contracts folder.
     */
-    browser
+    const browserName = browser.options.desiredCapabilities.browserName
+
+    return browserName === 'chrome' ? browser
```

Comment:
do you have any idea why it fails for firefox?

Reply:
I get this error `Unable to import "openzeppelin-solidity/contracts/math/SafeMath.sol": File not found` when the first test is disabled for firefox and chrome also.

Reply:
I don't get why the first test was disabled. was it introduced by parallel testing?

Reply:
No, i think it was introduced by you with these error messages.
```javascript
  var browserName = browser.options.desiredCapabilities.browserName
  if (browserName === 'safari' || browserName === 'internet explorer') {
    console.log('do not run remixd test for ' + browserName + ': sauce labs doesn\'t seems to handle websocket')
    browser.end()
    return
  }
  if (browserName === 'firefox') {
    console.log('do not run remixd test for ' + browserName + ': TODO to reenable later')
    browser.end()
    return
  }
```

Reply:
I have enabled remixd test for firefox and both tests now passes.